test/verify/check-terminal

```diff
@@ -193,6 +194,31 @@ PROMPT_COMMAND='printf "\\033]0;%s@%s:%s\\007" "${USER}" "${HOSTNAME%%.*}" "${PW
         b.wait_present('.terminal')
         b.wait_attr(".xterm-viewport", "style", white_style)
 
+    @nondestructive
+    def testOnlyTerminal(self):
+        b = self.browser
+        m = self.machine
+
+        m.write("/etc/cockpit/cockpit.conf", "[WebService]\nShell = /system/terminal.html\n")
```

Comment:
In this case m.write is okay as nondestructive tests always clean up /etc/cockpit/cockpit.conf (special case in testlib.py `nonDestructiveSetup()`).

Reply:
ah I didn't know that! neat